rotors_comm/msg/WindSpeed.msg

```diff
@@ -0,0 +1,6 @@
+Header header
+
+# Message should be expressed in the frame_id specified in the header.
```

Comment:
Since your not considering the header in the motor model, you might want to change this comment, or place a todo below.